src/main/java/fi/helsinki/ubipositioning/mqtt/MqttService.java

```diff
@@ -10,49 +10,64 @@
 import fi.helsinki.ubipositioning.datamodels.Observation;
 
 /**
- * MqttService.
+ * Implementation of interface to provide consumer possibility to
+ * get all the messages that have been sent to subscribed topic after start up
+ * and to publish data in messages into a another topic.
+ *
+ * @see IMqttProvider
  */
-public class MqttService implements IMqttService{
-    public static final int MAX_OBSERVATIONS = 10000;
+public class MqttService implements IMqttService {
+    /**
+     * Default buffer for storing data that comes from subscribed topic.
+     */
+    private static final int MAX_MESSAGES_TO_STORE = 10000;
+    /**
+     * Default lifetime for beacons.
```

Comment:
I think we could clarify that this is seconds.

Reply:
MAX_MESSAGES_TO_STORE or BEACON_LIFETIME?

Reply:
BEACON_LIFETIME